test/package/README.md

```diff
@@ -0,0 +1,62 @@
+# Package Testing
```

Comment:
Sorry, let's move this to these contributing docs: https://www.elastic.co/guide/en/kibana/current/development-tests.html

Reply:
Pushed with 66702b0